website/source/docs/drivers/raw_exec.html.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: "docs"
+page_title: "Drivers: Raw Exec"
+sidebar_current: "docs-drivers-raw-exec"
+description: |-
+  The Raw Exec task driver simply fork/execs and provides no isolation.
+---
+
+# Raw Fork/Exec Driver
+
+Name: `raw_exec`
+
+The `raw_exec` driver is used to execute a command for a task without any
+resource isolation. As such, it should be used with extreme care and is disabled
```

Comment:
s/resource isolation/isolation or resource constraints/ -- I think this is more consistent with phrasing we use elsewhere.

I would also add that `raw_exec` uses the same user permissions as the Nomad process. So if Nomad runs as root / administrator the forked process also runs as that user.